cpp/inline/mcts/SearchLog.inl

```diff
@@ -15,6 +15,19 @@ inline boost::json::object SearchLog<Game>::log_node_t::to_json() const {
   node_json["Q"] = Q_array;
 
   node_json["state"] = state;
+
+  boost::json::array provably_winning_array;
+  for (size_t i = 0; i < provably_winning.size(); ++i) {
+    provably_winning_array.push_back(provably_winning[i]);
```

Comment:
I would instead use `bitset_util::to_string()`, to make the strings more compact ("01" vs [false, true]).